updateQDKVersion.ps1

```diff
@@ -0,0 +1,45 @@
+$ver = $args[0]
+Write-Host $ver
+
+$pkgs=@(
+"Microsoft.Quantum.Standard",
+"Microsoft.Quantum.Development.Kit",
+"Microsoft.Quantum.Xunit",
+"Microsoft.Quantum.Katas",
+"Microsoft.Quantum.IQSharp",
+"Microsoft.Quantum.Development.Kit", 
+"Microsoft.Quantum.Chemistry",
+"Microsoft.Quantum.Chemistry.Jupyter",
+"Microsoft.Quantum.Compiler",
+"Microsoft.Quantum.IQSharp.Core",
+"Microsoft.Quantum.Numerics",
+"Microsoft.Quantum.Research",
+"Microsoft.Quantum.CsharpGeneration",
+"Microsoft.Quantum.Simulators")
+
+#Look at the first package and find its version
+$i = 0
+$csproj_string = 'Include="{0}" Version=' -f $pkgs[$i]
+Write-Host $csproj_string
+$file = (Get-ChildItem -recurse -Name -include "BasicGates/*.csproj")
+Write-Host $file
+$sentence = (Get-ChildItem -recurse | Select-String -pattern $csproj_string -Context 0).Line
+Write-Host (($sentence -split 'Version=')[1] | %{$_.split('"')[1]})
+$csproj_old = (($sentence -split 'Version=')[1] | %{$_.split('"')[1]})
+Write-Host  "Changing Version from $csproj_old to $ver"
+
+for .csproj files - <PackageReference Include="Microsoft.Quantum.???" Version="___" />
```

Comment:
PowerShell  doesn't recognize the lines starting with "for", which makes sense - they look like comments and not the actual PowerShell commands? My comment in the previous review was to change the logic of detecting the versions of the packages that need to be updated, from finding the old version in one project and replacing it across all other projects to finding all old versions in each of the contexts we have and replacing them with the new version, regardless of what the old version was.